app/views/admins/articles/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= simple_form_for article, as: :article, url: url, method: method do |f| %>
+  <% presenter = ArticlePresenter.new(f.object)%>
+  <% if method == 'patch' %>
+    <%= render partial: 'errors' %>
+  <% end %>
```

Comment:
Why only patch? What if there are errors when creating an article

Reply:
Oh, I forgot about this bug - with the Create action, validation errors are displayed near the inputs, but for some reason, with Update, they are not displayed, so I included partial _errors... This should be fixed, I'll google it.